### PR TITLE
Write unit tests for users sagas

### PR DESCRIPTION
**Pull request by ivan-bonk**

### CODE REVIEW

File: src/redux/users/users.saga.js [hunk 1]
```diff
@@ -133,15 +134,15 @@ function* handleTokenValidation({ payload }) {
   }
 }
 
-function* handleUsersError(e) {
+export function* handleUsersError(e) {
   yield put(setUsersLoading(false));
   yield put(setUserError({ e }));
   yield put(setSnackBarSeverity('error'));
-  yield put(setSnackBarMessage(e.message));
+  yield put(setSnackBarMessage(e.payload.message));
```
Comment (moran711) [inline src/redux/users/users.saga.js:141]: `e` object don't have `payload` property.

Comment (ivan-bonk) [reply src/redux/users/users.saga.js]: good point
thanks

**Review by moran711 — APPROVED** [review]

**Review by DmytroDidukh — APPROVED** [review]

File: jest.config.js [hunk 2]
```diff
@@ -5,10 +5,17 @@ module.exports = {
   modulePathIgnorePatterns: ['cypress'],
   coverageThreshold: {
     global: {
-      branches: 0,
```
Comment (ahd3r) [inline jest.config.js:8]: Please, rollback this file.

File: src/redux/users/tests/users-saga.test.js [hunk 3]
```diff
@@ -0,0 +1,314 @@
...
+        Users: {
+          ...mockUsersState,
+          user: mockUser
+        }
+      })
+      .run()
+      .then((result) => {
+        const { allEffects: analysis } = result;
+        const analysisPut = analysis.filter((e) => e.type === 'PUT');
+        expect(analysisPut).toHaveLength(3);
```
Comment (ahd3r) [inline src/redux/users/tests/users-saga.test.js:139]: Why you did not check the length of `CALL`?

File: jest.config.js (hunk 2 above)
Comment (ivan-bonk) [reply jest.config.js]: done

File: src/redux/users/tests/users-saga.test.js (hunk 3 above)
Comment (ivan-bonk) [reply src/redux/users/tests/users-saga.test.js:144]: good tip
I will add it

**Pull request merged**